docs/core/index.yml

```diff
@@ -0,0 +1,122 @@
+### YamlMime:Landing
+
+title: .NET Core documentation
+summary: Learn about .NET Core, an open-source, general-purpose development platform for building cross-platform apps.
+
+metadata:
+  title: .NET Core documentation
+  description: .NET Core is an open-source, general-purpose development platform for building cross-platform apps.
+  ms.topic: landing-page
+  ms.date: 03/24/2020
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+
+landingContent:
+
+  # Card
+  - title: Learn about .NET Core
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: About .NET Core
+            url: about.md
+          - text: NET Core SDK overview
+            url: sdk.md
+      - linkListType: whats-new
+        links:
+          - text: What's new in .NET Core 3.1
+            url: whats-new/dotnet-core-3-1.md
+          - text: What's new in .NET Core 3.0
+            url: whats-new/dotnet-core-3-0.md
```

Comment:
Let's get rid of this one. 3.0 is out of support now, so we don't need to highlight it. Perhaps replace it with 2.1 which is still in support.

Reply:
But the 3.0 what's new is effectively a 3.x what's new because not that much was added in 3.1.

Reply:
For reference, the 3.0 what's new page had 50% more page views in February than the 3.1 what's new page.

Reply:
Oh yeah you're right!